src/test/java/dev/aziz/movies/services/MovieServiceTest.java

```diff
@@ -50,4 +53,71 @@ void findMovieByIdTest() {
             assertEquals("James Cameron", foundMovie.getDirector());
         });
     }
+
+    @Test
+    void createMovieTest() {
+        // given
+        MovieDto movie = new MovieDto(4L, "The Dark Knight", "When the menace known as the Joker wreaks havoc and chaos on the people of Gotham, Batman must accept one of the greatest psychological and physical tests of his ability to fight injustice.\n", 2008, 5, "Christopher Nolan", List.of("Christian Bale", "Heat Ledger", "Aaron Eckhart", "Michael Caine", "Cillian Murphy"), "Action");
+
+        // when
+        MovieDto createdMovie = movieService.createMovie(movie);
+
+        // then
+        assertTrue(movieService.movieDtoList.contains(createdMovie));
```

Comment:
Instead of using the private field, compare the returned value with the input value.